backend/services/user_service.py

```
@@ -0,0 +1,528 @@
+
+            # Update user
+            new_balance = user.credits - 1
+            self.update_user(
+                email,
+                credits=new_balance,
+                credit_transactions=[t.model_dump(mode='json') for t in transactions],
+                total_jobs_created=user.total_jobs_created + 1
+            )
+
+            logger.info(f"Deducted 1 credit from {email} for job {job_id}. Remaining: {new_balance}")
+            return True, new_balance, f"Credit deducted. {new_balance} remaining"
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)